src/afl-fuzz.c

```diff
@@ -2512,7 +2512,8 @@ int main(int argc, char **argv_orig, char **envp) {
         if (unlikely(afl->is_main_node)) {
 
           if (unlikely(get_cur_time() >
-                       (afl->sync_time >> 1) + afl->last_sync_time)) {
+                       (afl->sync_time ? afl->sync_time : SYNC_TIME >> 1) +
```

Comment:
If we always set the `sync_time` on startup, after the initial env is read, we'd get rid of this extra branch, what do you think?